src/Template/Element/Form/core_properties.twig

```diff
@@ -1,18 +1,45 @@
 {% set attributes = data.attributes %}
+{% set module = currentModule.name|default('') %}
+{% set core_properties = config('Modules.' ~ module ~ '.core_attributes')|default(['title', 'description', 'uname', 'status']) %}
 
 <div class="tab"><h2>{{ __('Main Properties') }}</h2></div>
 
 <fieldset id="core_properties">
     {{ Form.hidden('id', {'value': object.id})|raw }}
-    {{ Form.control('title')|raw }}
-    {{ Form.control('description', {'type': 'textarea'})|raw }}
-    {{ Form.control('uname')|raw }}
-    {{ Form.control('status', {
-        'type': 'radio',
-        'options': [
-            {'value': 'on', 'text': __('On')},
-            {'value': 'draft', 'text': __('Draft')},
-            {'value': 'off', 'text': __('Off')},
-        ]
-    })|raw }}
+
+
+{% for key, property in core_properties %}
+    {% if property == 'password' %}
+    <div class="password-component">
+        {{ Form.control('password', {
+            'class': 'password',
+            'placeholder': 'new password',
+            'autocomplete': 'new-password',
+            'default': ''
+        })|raw }}
+        {{ Form.control('password', {
+            'label': 'Retype password',
+            'id': 'confirm_password',
+            'name': 'confirm-password',
+            'class': 'confirm-password',
+            'placeholder': 'confirm password',
+            'autocomplete': 'new-password',
+            'default': ''
+        })|raw }}
+    </div>
+    {% elseif property == 'description' %}
+        {{ Form.control('description', {'type': 'textarea'})|raw }}
+    {% elseif property == 'status' %}
+        {{ Form.control('status', {
+            'type': 'radio',
+            'options': [
+                {'value': 'on', 'text': __('On')},
+                {'value': 'draft', 'text': __('Draft')},
+                {'value': 'off', 'text': __('Off')},
+            ]
+        })|raw }}
+    {% else %}
+        {{ Form.control(property, {'type': Schema.getControlTypeFromSchema(property)})|raw }}
```

Comment:
This `Schema.getControlTypeFromSchema()` was supposed to do exactly the job that's being done here in this template using a series of `if / else`. Could this logic be moved there instead?

Reply:
mhm, not sure. `getControlTypeFromSchema` return a string type, it doesn't handle "view" logic.
However I'll give it a deeper look, trying to use it, instead of ugly `if / else`

Reply:
Actually this doesn't solve all the cases. Specifically, the "password + password confirm" is a bit of an edge case. But for status and description should work.

The "password + confirm password" can be probably combined into a single control, but this requires additional configuration for CakePHP's `FormHelper`, and it's probably too much beyond the scope of this PR.

Reply:
(Return type of the helper method can be widened to "string|array", so that in some cases, like "status", an associative map is returned.)